src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -30,9 +30,20 @@ Amber::Server.configure do |settings|
   #
   #
   # Colorize Logging: specifies whether or not to use ANSI color codes
-  # when logging information. Defaults to true.
+  # when logging information, display the time and/or to display the severity level.
+  # Defaults to true.
   #
-  # settings.colorize_logging = true
+  # Log Level defines the verbosity of the Amber logger. This option defaults to
+  # debug for all environments. The available log levels are: debug, info, warn,
+  # error, fatal, and unknown.
```

Comment:
In this comment, it seems that it is talking about `severity` and not `level` (as coded).

Do we want `severity` to be called `level` and `level` to be called `output_log_level`?

Reply:
I thought of prefixing the boolean fields with `show_` so it would read

show_color
show_time
show_level
level
skip
context
filter

Reply:
Since level is an abstraction that indicates how severities relate to each other for the purpose of stream filtering, I think a log _message_ has a severity and a log stream has a severity level filter, aka `log.level`.

Choosing what to print for a given log message would be `show_severity` and choosing which logs to filter out would be `level`.

Rails config doesn't nest logger settings in a logging namespace, but the parallel is here:

> config.log_level defines the verbosity of the Rails logger.

As far as I can tell, Rails doesn't provide the configuration option to prevent display of  timestamps and severity without implementing your own log formatter class, though they do provide an option called `config.colorize_logging`. 

Since there is a format string option, is providing the boolean flags `show_time` and `show_level` or `show_severity` redundant?